.github/workflows/test.yml

```diff
@@ -269,16 +276,58 @@ jobs:
   test-spell-check:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
         with:
-          repository: FreeRTOS/coreMQTT
-          ref: main
+          repository: skptak/coreMQTT
```

Comment:
fix this repo?

Reply:
Requires a wordlist that doesn't exist. The PR for coreMQTT using this action can be found [here](https://github.com/FreeRTOS/coreMQTT/pull/260)